bootstrap/src/components/Header/Header.tsx

```diff
@@ -1,25 +1,27 @@
 import React, { useState, useEffect, useCallback } from "react";
 import throttle from "lodash/throttle";
-import Logo from "./Logo";
 
-import styles from "./Header.module.scss";
-import Links from "./Links";
-import Aside from "./Aside";
 import { useAuth } from "../../contexts/Auth";
 import { AuthStatus } from "../../contexts/Auth/state";
 // uncomment the next line to use WorkspacePicker
 // import WorkspacePicker from "../controls/WorkspacePicker";
 
-const VR = ({ ...restProps }) => <div className={styles.VR} {...restProps} />;
+import Logo from "./Logo";
+import Links from "./Links";
+import Aside from "./Aside";
+
+import styles from "./Header.module.scss";
+
+const VR: React.FC = ({ ...restProps }) => <div className={styles.VR} {...restProps} />;
```

Comment:
what does VR stand for?

Reply:
I did not know either and considered renaming it. It is "vertical line delimiter", i.e., line between picker and tabs. I can rename it to `VerticalDelimiter` or something similar.

Reply:
Fixed.